intuita-webview/src/jobDiffView/DiffViewer/Header/index.tsx

```diff
@@ -48,19 +59,22 @@ const Header = ({ title, viewType, jobs, onViewChange }: Props) => {
 				</VSCodeButton>
 			</div>
 			<div className={styles.actionsContainer}>
-				<VSCodeButton
-					appearance="primary"
-					onClick={handleSaveToFileSystem}
-				>
-					Save to File System
-				</VSCodeButton>
-				<VSCodeButton
-					appearance="primary"
-					title="Show commit options"
-					onClick={handleCommit}
-				>
-					Commit...
-				</VSCodeButton>
+				{changesAccepted ? (
+					<VSCodeButton
+						appearance="primary"
+						title="Show commit options"
+						onClick={handleCommit}
+					>
+						Commit...
+					</VSCodeButton>
+				) : (
+					<VSCodeButton
+						appearance="primary"
+						onClick={handleApplySelected}
+					>
+						Apply selected
```

Comment:
I like `Apply all`.. but this is debatable..

Reply:
`Apply selected` looks better to me, because we will be able to select specific jobs to be applied

Reply:
Ahh, in that case, yes, agree